webapp/channels/src/components/discard_changes_modal.tsx

```diff
@@ -12,24 +12,23 @@ type Props = {
     onCancel: (checked: boolean) => void;
 }
 
-export default class DiscardChangesModal extends React.PureComponent<Props> {
-    public render(): JSX.Element {
-        const title: JSX.Element = (
+const DiscardChangesModal = ({show, onConfirm, onCancel}: Props) => {
```

Comment:
Just to follow the pattern we follow in other files, put each argument in a different line:

```suggestion
const DiscardChangesModal = ({
    show,
    onConfirm,
    onCancel,
}: Props) => {
```

This makes it easier in the future to recognize in a glance what props have been added / removed by a change.